data/json/items/comestibles/med.json

```diff
@@ -1677,6 +1677,24 @@
     "flags": [ "IRREPLACEABLE_CONSUMABLE" ],
     "charges": 10,
     "stack_size": 10
+  },
+    {
+    "id": "sugar_pills",
+    "type": "COMESTIBLE",
+    "comestible_type": "MED",
+    "name": "homeopathic pill",
+    "description": "Some homeopathic pills, prepared through the repeated dilution of an active ingredient. The label purpots that the pills are able to treat flu-like symptoms and maintain general wellness.",
+    "weight": "1 g",
+    "volume": "100 ml",
+    "price": 1000
+    "price_postapoc": 1,
+    "fun": 1,
+    "charges": 20,
+    "stack_size": 20,
+    "flags": [ "IRREPLACEABLE_CONSUMABLE" ],
```

Comment:
I think this flag means it's supposed to be high value because we can't make it anymore. I'm not sure those are really not possible to make post cataclysm.

Reply:
Good point.

Reply:
While you're technically correct, some homeopathic preparations are very complex and difficult to represent in CDDA, due to the sheer ratio of water to the active ingredient. I think the flag should stay unless someone makes a recipe for it.

Reply:
But it's pretty easy to just lie about the preparation.